Cargo.toml

```diff
@@ -85,6 +85,7 @@ server = ["dep:httpdate", "dep:pin-project-lite", "dep:smallvec"]
 
 # C-API support (currently unstable (no semver))
 ffi = ["dep:http-body-util", "futures-util?/alloc"]
+capi = []
```

Comment:
Probably would be nice to add here the required features `client,http1,http2,ffi`

Reply:
Yea, the reason I haven't added them there automatically is because it's possible to choose to only enable http1 or http2, and there's a separate PR here as some people are working on the server portion of the C API.